start.py

```diff
@@ -805,6 +801,40 @@ def start_app(m2ee):
         logger.warning('start failed, stopping')
         sys.exit(1)
 
+@atexit.register
+def terminate_process():
+    logger.warning('Stopping M2EE ...')
+    m2ee.stop()
+    logger.warning('Terminating all child processes ...')
+    try:
+        this_process = psutil.Process()
+        child_processes = this_process.children(recursive=True)
+        terminate_child_processes(child_processes)
+        logger.info('SIGTERM sent to all child processes.')
+        terminated_children, running_children = psutil.wait_procs(child_processes, timeout=3)
+        logger.info("Terminated {} child processes.".format(len(terminated_children)))
```

Comment:
Log lines on info will show up every single time a process is stopped. We should not bother the users with this. For debugging purposes while developing you can use the `BUILDPACK_XTRACE` environment variable.